presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -448,6 +448,17 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kDriverMaxPagePartitioningBufferSize{
       "driver.max-page-partitioning-buffer-size"};
 
+  /// Enable TTL for AsyncDataCache and SSD cache.
+  static constexpr std::string_view kCacheVeloxTtlEnabled{
+      "cache.velox.ttl-enabled"};
+  /// TTL duration for AsyncDataCache and SSD cache entries.
+  static constexpr std::string_view kCacheVeloxTtlThreshold{
```

Comment:
Let's remove velox and name it retention
kAsyncDataCacheTtlRetentionPeriod

And let's move these configs close to AsyncDataCache configs.

Reply:
Presto (java) also has these configs, and, agreed with Sergey's suggestion, we want to match and in the above way.

Reply:
Yeah I'm a bit suspicious on the purpose of matching Java if these are not passed over by Java coordinator. The downside of matching Java here is clear though: We are making C++ configs more confusing. As someone who does not understand fully on C++ codebase, he might not relate directly these configs with async data cache while it is actually directly related IMO.

Reply:
I tend to want to match Java, but don't have strong opinion. It happens to us quite often that we look for what value Presto Java or Trino are using and how they works. So matching the config name gives ppl confidence that the config is what they're looking for. btw, this TTL applies both to async data cache and ssd.

Reply:
AsyncData cache is a cache wrapper umbrella that takes ram as first level and ssd as second. So if ttl applies to async data cache then it should semantically be applied to both ram and ssd.